Header header
geometry_msgs/Pose poseTip2
geometry_msgs/Pose poseLink6
geometry_msgs/Pose poseTip1
